src/MakersOfDenmark.Application/Commands/V1/admin/EditBaseMakerSpace.cs

```diff
@@ -0,0 +1,45 @@
+﻿using MakersOfDenmark.Domain.Enums;
+using MakersOfDenmark.Infrastructure.Persistence;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MakersOfDenmark.Application.Commands.V1.admin
+{
+    public class EditBaseMakerSpace : IRequest
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public string VATNumber { get; set; }
+        public string LogoUrl { get; set; }
+        public string AccessType { get; set; }
+    }
+
+    public class EditBaseMakerSpaceHandler : IRequestHandler<EditBaseMakerSpace>
+    {
+        private readonly MODContext _context;
+
+        public EditBaseMakerSpaceHandler(MODContext context)
+        {
+            _context = context;
+        }
+        public async Task<Unit> Handle(EditBaseMakerSpace request, CancellationToken cancellationToken = default)
+        {
+            var makerSpace = await _context.MakerSpace.FirstOrDefaultAsync(x => x.Id == request.Id);
+            if (makerSpace == null)
```

Comment:
makerSpace is null er bedre at bruge her.

På den måde slipper vi for operator overloading, og tjekker kun på referencen.